codegens/http/lib/util.js

```diff
@@ -168,7 +168,8 @@ function getHeaders (request) {
     headers = '';
 
   if (contentTypeIndex >= 0) {
```

Comment:
The fix is inside this particular block which checks if the 'Content-Type' header is present.  
The unit test you have written does not pass this criterion of `contentTypeIndex >= 0`. Hence the fix is not being tested. That is why the initial typo wasn't detected.
@shreys7

Reply:
yes. Shall I change the tests?

Reply:
Yes, you will have to add a 'Content-Type' header. Only then the unit test make sense.